spec/spec_helper.rb

```diff
@@ -12,6 +12,14 @@
 # the additional setup, and require it from the spec files that actually need
 # it.
 #
+
+if ENV['CI']
+  require 'simplecov'
+  SimpleCov.start do
+    add_filter "/spec/"
+  end
+end
```

Comment:
> not tested but copied from manageiq

Probably better to copy from another gem as opposed to an application.

So, here it would just be (I looked at linux_admin as an example)

```suggestion
if ENV['CI']
  require 'simplecov'
  SimpleCov.start
end
```

Reply:
yea. saw no need to run simplecov on specs, since they always display 100%

Reply:
I was certain I had a script for new repositories and ran it on here.  It should have included all this 😕